gradle/buildSrc/src/main/kotlin/io/spine/gradle/internal/IncrementGuard.kt

```diff
@@ -43,4 +55,15 @@ class IncrementGuard : Plugin<Project> {
             shouldRunAfter("test")
         }
     }
+
+    /**
+     * Returns `true` if the current build is a Travis job which represents a GitHub pull request.
+     *
+     * Implementation note: the `TRAVIS_PULL_REQUEST` environment variable contains the pull
+     * request number rather than `"true"` in positive case, hence the check.
+     */
+    private fun isTravisPullRequest(): Boolean {
```

Comment:
Can you please add a link to the Travis env vars docs? ( https://docs.travis-ci.com/user/environment-variables/#default-environment-variables )